oecophylla/taxonomy/taxonomy.rule

```diff
@@ -84,13 +84,12 @@ rule taxonomy_kraken:
         forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
         reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
     output:
-        map = taxonomy_dir + "{sample}/kraken/{sample}_map.txt",
-        report = taxonomy_dir + "{sample}/kraken/{sample}_report.txt",
+        map = taxonomy_dir + "{sample}/kraken/{sample}.map.txt",
+        report = taxonomy_dir + "{sample}/kraken/{sample}.report.txt",
     params:
-        kraken_env = config['envs']['kraken'],
-        kraken_db = config['params']['kraken']['kraken_db']
-    threads:
-        12
+        env = config['envs']['kraken'],
+        db = config['params']['kraken']['db'],
+        threads = config['params']['kraken']['threads']
```

Comment:
Threads should be specified as a separate entry at the same level as 'params'... see other rules for example.

Reply:
Okay I looked at other rules. They have `threads` in the rule file but not in the `config.yaml` file. I guess that these `threads` are default settings and can be overridden by `cluster.json`. Is that correct?